docker-compose.yml

```
@@ -15,7 +15,7 @@ secrets:
 
 services:
   mailer:
-    image: 'dhsncats/cyhy-mailer:1.4.1'
+    image: 'dhsncats/cyhy-mailer:1.4.2'
```

Consistent with the other version suggestion.

```suggestion
    image: 'dhsncats/cyhy-mailer:1.5.0'
```